source/_integrations/homekit.markdown

```diff
@@ -584,7 +596,7 @@ A doorbell sensor can be linked via the `linked_doorbell_sensor` configuration s
 
 #### HomeKit stalls or devices respond slowly with many cameras
 
-HomeKit updates each camera snapshot sequentially when there are multiple cameras on a bridge. The HomeKit update methodology can lead to the app stalling or taking a while to update. To avoid this problem, limit each `HomeKit Bridge` to 6 cameras and create a new `HomeKit Bridge` for additional cameras.
+HomeKit camera snapshots tie up the HomeKit connection while the snapshot is being taken. To avoid this problem, create a seperate `HomeKit` instance in [Accessory Mode](#mode) for each camera.
```

Comment:
```suggestion
HomeKit camera snapshots tie up the HomeKit connection during snapshots. To avoid this problem, create a separate `HomeKit` instance in [Accessory Mode](#mode) for each camera.
```